src/utils/helpers/evaluateElements.ts

```diff
@@ -76,7 +76,7 @@ const evaluateSingleElement: EvaluateElement = async (
         ? await evaluateExpression(expressionOrValue, evaluationParameters)
         : expressionOrValue
     } catch (e) {
-      console.log(e, expressionOrValue)
+      // console.log(e, expressionOrValue)
```

Comment:
This has been annoying me for while in the console, so just suppressed it for now. Would like to do a bit more debugging in here at some point, but for now it's easier to ignore in the console.